From 745e53620b54025df94d206668f76429b0955b1f Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 5 Apr 2019 13:13:24 +0000
Subject: [PATCH] Revert "abuild: unset depends for subpackages"

Apparently there are many packages that does soemthing like:

subpackages="$pkgname-foo:_foo"

_foo() {
	depends="$depends something-else"
}

and thus depend on the previous behavior. We need to revert and plan
this better.

This reverts commit 8fbbffd201a28a06804c7f6d3a2b5cd948c6ce07.
---
 abuild.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index d19d12e..1222cb7 100644
--- a/abuild.in
+++ b/abuild.in
@@ -756,7 +756,6 @@ pre_split() {
 		return 0
 	fi
 	# the subpackages should not inherit those from main package
-	depends=""
 	provides=""
 	install_if=""
 }
-- 
2.21.0

